{"name": "test_huggingface_model_loading", "status": "passed", "description": "Test HuggingFace model loading error.", "attachments": [{"name": "log", "source": "c9659d12-4ebc-4eac-bd89-76eb6cfabb56-attachment.txt", "type": "text/plain"}], "start": 1757657445215, "stop": 1757657445216, "uuid": "875b86e4-1733-420a-a550-0a1f153397c1", "historyId": "c057c168d18753251139580dae16491b", "testCaseId": "c057c168d18753251139580dae16491b", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_huggingface_model_loading", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "59211-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}